sql/core/src/main/scala/org/apache/spark/sql/DataFrame.scala

```diff
@@ -888,6 +888,21 @@ class DataFrame private[sql](
   }
 
   /**
+   * Returns a new [[DataFrame]] without duplicates.
+   * @group dfops
+   */
+  def dropDuplicates(): DataFrame = dropDuplicates(this.columns)
+
+  /**
+   * Returns a new [[DataFrame]] without duplicates under the given columns.
+   * @group dfops
+   */
+  def dropDuplicates(subset: Seq[String]): DataFrame = {
```

Comment:
@viirya No, dropDuplicates is used to remove duplicate rows that are the same in some columns or in all columns (default) . The default version is as same as distinct.